src/app/home/home.component.ts

```diff
@@ -19,12 +19,11 @@ export class HomeComponent implements OnInit {
         filter((msg: EventMessage) => msg.eventType === EventType.LOGIN_SUCCESS),
       )
       .subscribe((result: EventMessage) => {
-        console.log(result);
         const payload = result.payload as AuthenticationResult;
         this.authService.instance.setActiveAccount(payload.account);
+        this.setLoginDisplay();
```

Comment:
```suggestion
        this.setLoginDisplay();
```
do you need to call setLoginDisplay() here?

Reply:
Yes we do, After `LOGIN_SUCCESS` the component doesn't update, so we need to call  `setLoginDisplay` again after `LOGIN_SUCCESS`.